docs/docs/10.4-test-utils.md

```diff
@@ -16,15 +16,22 @@ Simulate.{eventName}(DOMElement element, object eventData)
 
 Simulate an event dispatch on a DOM node with optional `eventData` event data. **This is possibly the single most useful utility in `ReactTestUtils`.**
 
-Example usage:
+**Clicking an element**  
+```javascript
+var node = React.findDOMNode(this.refs.button);
+React.addons.TestUtils.Simulate.click(node);
+```
 
+**Changing the value of an input field and then pressing ENTER**  
 ```javascript
 var node = React.findDOMNode(this.refs.input);
-React.addons.TestUtils.Simulate.click(node);
-React.addons.TestUtils.Simulate.change(node, {target: {value: 'Hello, world'}});
-React.addons.TestUtils.Simulate.keyDown(node, {key: "Enter"});
+node.value = 'giraffe'
```

Comment:
Well, unless you have something particular like a two way binding, you usually have an event handler attached to your inputs, right?

E.g. This snippet doesn't work with `event.target.value`  (code [here](https://github.com/dmatteo/simulate-test.git))
```javascript
  it('should add elements to .state.values on "Enter"', function() {
    var instance = TestUtils.renderIntoDocument(<SomeComponent />);
    var inputNode = React.findDOMNode(instance.refs.inputField);

    //inputNode.value = 'hello';
    Simulate.change(inputNode, {target: {value: 'hello'}});
    Simulate.keyDown(inputNode, {keyCode: 13, which: 13, key: 'Enter'});

    inputNode.value = 'react';
    Simulate.change(inputNode);
    Simulate.keyDown(inputNode, {keyCode: 13, which: 13, key: 'Enter'});

    expect(instance.state.values, 'to exhaustively satisfy', ['hello', 'react']);
  });
```

Don't you think that is a source of confusion in how Simulate works?